ui/src/pageLayout/containers/Nav.tsx

```diff
@@ -89,6 +90,9 @@ class SideNav extends PureComponent<Props, State> {
     const feedbackLink =
       'https://docs.google.com/forms/d/e/1FAIpQLSdGJpnIZGotN1VFJPkgZEhrt4t4f6QY1lMgMSRUnMeN3FjCKA/viewform?usp=sf_link'
 
+    // Cloud
+    const cloudUsersLink = `${CLOUD_URL}/users`
```

Comment:
making `/users` it's own const would make it more uniform with the convention for quartz urls like `/usage` and `/billing` and `/logout`